src/root_files/app/localstorage-sync.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>Deriv</title>
+        <meta charset='utf-8' />
+        <meta name='referrer' content='origin' />
+        <script>
+            window.addEventListener('message', function(message) {
+                var restricted_origins = [
+                    'https://deriv.app', 
+                    'https://staging.deriv.app'
+                ];
+
+                // TODO: Remove always true condition.
+                if (true || restricted_origins.includes(message.origin)) {
```

Comment:
why call it restricted_origins?

Reply:
Cause messages should only be processed if coming from origins contained within `restricted_origins`.

Reply:
could we then instead call it allowed_origins? :D